packages/vant/src/action-sheet/ActionSheet.tsx

```diff
@@ -94,6 +95,12 @@ export default defineComponent({
       }
     };
 
+    const renderIcon = (action: ActionSheetAction) => {
+      if (action.icon) {
+        return <Icon class={bem('item--icon')} name={action.icon} />;
```

Comment:
icon 对应 BEM 里的 element，所以不应该使用修饰符，直接中划线分隔就可以哈

```suggestion
        return <Icon class={bem('item-icon')} name={action.icon} />;
```

Reply:
Done